networks/__init__.py

```diff
@@ -0,0 +1,12 @@
+import torch.nn as nn
+
+from networks.caml import CAML
+from networks.kim_cnn import KimCNN
+from networks.xml_cnn import XMLCNN
+
+
+def get_init_weight_func(config):
+    def init_weight(m):
+        if isinstance(m, nn.Linear) or isinstance(m, nn.Conv2d):
```

Comment:
but now we do Conv1d?

Reply:
It seems that Si-An has added `nn.Conv1d` here locally. @sian-chen

Reply:
Yes, I've added Con1d here.